scripts/generate_devices.py

```diff
@@ -83,21 +84,16 @@ def __hash__(self) -> int:
         name = device["typeName"]
 
         if not name.startswith("Matter "):
-            print("Unexpected: device doesn't start with Matter. Skipping")
```

Comment:
We don't need to remove prints from scripts if we think the prints are useful.

Reply:
yeah scripts/examples may have prints